src/core/agent/modules/signify/signifyApi.ts

```diff
@@ -390,4 +393,19 @@ export class SignifyApi {
       name: name,
     };
   }
+
+  async rotateIdentifier(signifyName: string): Promise<void> {
+    const rotateResult = await this.signifyClient
+      .identifiers()
+      .rotate(signifyName);
+    let operation = await rotateResult.op();
+    operation = await this.waitAndGetDoneOp(
+      operation,
+      this.opTimeout,
+      this.opRetryInterval
+    );
+    if (!operation.done) {
+      throw new Error(SignifyApi.FAILED_TO_ROTATE_AID);
+    }
+  }
```

Comment:
@iFergal What we return for this function?

Reply:
I think `void` is OK for now